Shared/ContentView.swift

```diff
@@ -8,14 +8,22 @@
 import SwiftUI
 
 struct ContentView: View {
+    @EnvironmentObject var user: UserViewModel
+    
     var body: some View {
-        Text("Hello, Fun Gen!")
-            .padding()
+        NavigationView {
+            if user.userIsAuthenticatedAndSynced {
```

Comment:
It looks like even if the user has logged in, restarting the app will put the user in the AuthenticationView. You might want to (in UserViewModel)

1. sync the user on init or switch to use snapshot listener
2. potentially need to [Listen for authentication state](https://firebase.google.com/docs/auth/ios/start#listen_for_authentication_state), but I'm not sure

Reply:
Would they no longer be authenticated if the app was restarted?

Reply:
If you change the code to this:

```suggestion
            if user.userIsAuthenticated {
```

You'll notice that it takes you to the profile view instead 🤯

Reply:
Do note that changing just this is probably not enough, as UserViewModel doesn't have the current user's info synced

Reply:
This change will keep a user logged in after a force quit of the app. Should be good enough for now. I agree it won't be enough, but this is work in progress. Firestore database needs to be setup this week